handbook/engineering/onboarding.md

```diff
@@ -13,6 +13,7 @@ Welcome to Sourcegraph! This document will guide you through engineering specifi
   - [Opsgenie](https://sourcegraph.app.opsgenie.com/settings/users/)
   - [Docker Hub](https://hub.docker.com/orgs/sourcegraph)
   - [Site24x7](https://www.site24x7.com) (optional; recommended for Distribution team members)
+  - [HoneyComb.io](optional; recommended for Distribution team members)
```

Comment:
Is there any reasons why every engineer shouldn't have access to this? Do we pay per user?

Reply:
It came up in a meeting with Beyang. I'm not sure if the pricing is per user.

Reply:
Let's not make it optional (and I also fixed the link)

```suggestion
  - [HoneyComb.io](https://www.honeycomb.io/)
```